mmpose/visualization/local_visualizer.py

```diff
@@ -243,6 +248,20 @@ def _draw_instances_kpts(self,
             else:
                 keypoints_visible = [np.ones(len(kpts)) for kpts in keypoints]
 
+            if skeleton_style == 'openpose':
+                neck = (keypoints[:, 5] + keypoints[:, 6]) / 2
+                neck_score = (scores[:, 5] + scores[:, 6]) / 2
```

Comment:
Regarding the confidence score for the `neck`, I think it should be considered based on different situations:
1. When both `left_shoulder` and `right_shoulder` are present (score >= args.kpt_thr), the average score of the two can be used as the score for the neck. 
3. When one shoulder is missing, the score for the neck should be 0

Reply:
BTW, can you also fix this in `projects/mmpose4aigc/openpose_visualization.py` this [line](https://github.com/open-mmlab/mmpose/blob/71d74de89cdd87f147f0b091e65f07b5fe6450f0/projects/mmpose4aigc/openpose_visualization.py#L67)?